website/docs/r/search_index.html.markdown

```diff
@@ -28,7 +28,25 @@ resource "mongodbatlas_search_index" "test-basic-search-index" {
 }
 ```
 
-### Advanced (with custom analyzers)
+### Basic vector index
+```terraform
+resource "mongodbatlas_search_index" "test-basic-search-vector" {
```

Comment:
@Zuhairahmed do we want a different example?

Reply:
@lantoli  suggest for time being we leverage example that Atlas Search team has provided here: https://www.mongodb.com/docs/atlas/atlas-search/field-types/knn-vector/#std-label-fts-data-types-knn-vector

Reply:
"fields": {
      "plot_embedding": {
        "type": "knnVector",
        "dimensions": 1536,
        "similarity": "euclidean"
      }
    }

Reply:
also believe you are missing `name` parameter which is required: 
<img width="555" alt="image" src="https://github.com/mongodb/terraform-provider-mongodbatlas/assets/47370943/d011d9f3-5c43-4181-a0b4-5bd64169bc58">

Reply:
thanks. your example won't work directly but adapting the idea.  current "name" attribute is the index name, that doesn't change.

Reply:
SGTM